Example/Podfile

```diff
@@ -3,6 +3,7 @@ platform :ios, '13.0'
 
 target 'StylableSwiftUI_Example' do
   pod 'StylableSwiftUI/Animated', :path => '../'
+  pod 'SwiftLint'
```

Comment:
We care which version, or just always the latest?

Reply:
I'm quite happy with just latest since the actual version will be in the lock file so it will only be updated if we manually run pod update - but if you feel strongly the other way I'll put the version on it